src/components/learner-credit-management/assignment-modal/NewAssignmentModalButton.jsx

```diff
@@ -114,11 +133,11 @@ const NewAssignmentModalButton = ({ enterpriseId, course, children }) => {
       trackEventMetadata,
     );
   };
-
   const handleAllocateContentAssignments = () => {
+    // If no assignmentRun key exist, fall back to the top level course key
```

Comment:
[nit/clarification] Is this still a valid comment? It looks like we're always passing `assignmentRun.key` without a fallback (which I believe is intentional?).